package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "immortalityidle",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
Please don't manually increase the version number, my build script takes care of that.

Reply:
Ok, that's good to know. I changed it back to 1.1.1 and just reverted the package-lock.json since I assume the build script probably handles that too.